.github/fabricbot.json

```diff
@@ -2863,137 +2863,1609 @@
       ]
     }
   },
-    {
-        "taskSource": "fabricbot-config",
-        "taskType": "trigger",
-        "capabilityId": "IssueResponder",
-        "subCapability": "IssuesOnlyResponder",
-        "version": "1.0",
-        "config": {
-            "conditions": {
-                "operator": "and",
+  {
```

Comment:
Not sure where all of these changes are coming from, seems like VS is formatting the file. I'll fix this.

Reply:
I recommend just running `merge.js` from the command line without any IDEs open.